interface/weno/adminClass.php

```diff
@@ -0,0 +1,723 @@
+<?php
```

Comment:
This comment is in the interface/weno/adminClass.php script

Need license/author header.